provider/calc.go

```diff
@@ -102,6 +111,21 @@ func (o *calcProvider) floatGetter() (float64, error) {
 			res += v
 		}
 
+	case len(o.div) > 0:
+		for idx, p := range o.div {
+			v, err := p()
+			if err != nil {
+				return 0, fmt.Errorf("div[%d]: %w", idx, err)
+			}
+			if res == 0 {
```

Comment:
was machst Du wenn res beim zweiten Operand immer noch Null ist? Die Logik erscheint fehlerhaft?

Reply:
Stimmt :). Braucht aber einen kleinen Kommentar. Und Div/0 müsste vmtl auch abgefangen oder wenigstens mal getestet werden.

Reply:
If the dividend is 0 then the result is 0 right away, otherwise the value will be divided by every divisor != 0. Divisor == 0 are ignored and don't change the result